go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/pkg/errors v0.9.1
 	github.com/sourcegraph/campaignutils v0.0.0-20201124055807-2f9cfa9317e2
-	github.com/sourcegraph/codeintelutils v0.0.0-20210113171425-9ec641b48a8e
+	github.com/sourcegraph/codeintelutils v0.0.0-20210118225004-6e6891d301ca
```

Comment:
Can we bump this to a new commit if we handle https://github.com/sourcegraph/codeintelutils/pull/10#discussion_r559824707? (I think it was misread, we should call it `associatedIndexId` instead of `associatedIndexID` to be consistent with the existing parameters.